index.html

```diff
@@ -25,56 +30,112 @@
+          <h4 class="col-12">@ la recta de Azua, Dominican Republic</h4>
+        </div>
+      </div>
     </header>
     <main class="row main">
       <section class="col main-program">
-        <div>
+        <div class="program-heading">
           <h2>Main Program</h2>
-          <div class="row">
-            <div class="col-sm-2"><i class="fas fa-tachometer-alt"></i></div>
-            <div class="col-sm-3"><p>Tuning</p></div>
-            <div class="col-sm-7"><p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p></div>
+          <span></span>
+        </div>
+        <div class="col-12">
+          <div class="row activities">
+            <div class="col-11 col-md-2 activity">
+              <div class="row activity-data">
+                <div class="col-1 col-md-6"><i class="fas fa-tachometer-alt"></i></div>
+                <div class="col-4 col-md-11 activity-name"><h5>Tuning</h5></div>
+                <div class="col-6 col-md-11 activity-description"><p>Lorem ipsum dolor sit amet.</p></div>
+              </div>
+            </div>
+            <div class="col-11 col-md-2 activity">
+              <div class="row activity-data">
+                <div class="col-1 col-md-6"><i class="fas fa-clock"></i></div>
+                <div class="col-4 col-md-11 activity-name"><h5>Endurance</h5></div>
+                <div class="col-6 col-md-11 activity-description"><p>Lorem ipsum dolor sit amet.</p></div>
+              </div>
+            </div>
+            <div class="col-11 col-md-2  activity">
+              <div class="row activity-data">
+                <div class="col-1 col-md-6"><i class="fas fa-mountain"></i></div>
+                <div class="col-4 col-md-11 activity-name"><h5>Of-Road</h5></div>
+                <div class="col-6 col-md-11 activity-description"><p>Lorem ipsum dolor sit amet.</p></div>
+              </div>
+            </div>
+            <div class="col-11 col-md-2  activity">
+              <div class="row activity-data">
+                <div class="col-1 col-md-6"><i class="fas fa-stopwatch"></i></div>
+                <div class="col-4 col-md-11 activity-name"><h5>Dragstrip</h5></div>
+                <div class="col-6 col-md-11 activity-description"><p>Lorem ipsum dolor sit amet.</p></div>
+              </div>
+            </div>
+            <div class="col-11 col-md-2 activity">
+              <div class="row activity-data">
+                <div class="col-1 col-md-6"><i class="fas fa-tachometer-alt"></i></div>
+                <div class="col-4 col-md-11 activity-name"><h5>Tuning</h5></div>
+                <div class="col-6 col-md-11 activity-description"><p>Lorem ipsum dolor sit amet.</p></div>
+              </div>
+            </div>
+          </div>
+          <div class="act-button">
+            <button>Get Your Entry</button>
           </div>
         </div>
       </section>
-      <section class="col-sm-12 pilots">
```

Comment:
- [ ] **The overall design of your pilot section is significantly different from the original. Please fix it**

Original Design|Cloned Desing
:--------------------:|:-----------------:
![image](https://user-images.githubusercontent.com/59168430/124210175-9a4eaa00-db08-11eb-9ae9-9618b40dc903.png)|![image](https://user-images.githubusercontent.com/59168430/124210244-b5b9b500-db08-11eb-9575-1bb703434835.png)

Things like image size and spacing between images and text, don't match the original design. Please fix this to make it similar to the original design.